atlasdb-commons/src/main/java/com/palantir/exception/NotInitializedException.java

```diff
@@ -16,18 +16,40 @@
 package com.palantir.exception;
 
 import com.palantir.common.exception.AtlasDbDependencyException;
+import com.palantir.logsafe.Arg;
+import com.palantir.logsafe.Safe;
 import com.palantir.logsafe.SafeArg;
+import com.palantir.logsafe.SafeLoggable;
+import java.util.List;
 
-public class NotInitializedException extends AtlasDbDependencyException {
-    public NotInitializedException(String objectNotInitialized) {
-        super(exceptionMessage(objectNotInitialized));
+public final class NotInitializedException extends AtlasDbDependencyException implements SafeLoggable {
```

Comment:
Have you verified that this is safe to make final? (that is, there's no one who subclasses NIE) - this is on the public API so it's not necessarily safe to do this